README.adoc

```diff
@@ -1029,6 +1029,20 @@ User.all.size
 User.all.length
 ----
 
+=== `where.not` with multiple attributes
+
+Until Rails 6.1, when negating condition with multiple attributes using `where.not`, it works like `(NOT(A) AND NOT(B))`. This does not follow boolean algebra law `NOT (A AND B) = (NOT(A) OR NOT(B))` and could lead to confusing results.
```

Comment:
Until Rails 6.0?
https://github.com/rails/rails/pull/36029

Reply:
`6.0` added a deprecation warning but it still worked like NOR. The switch happened in `6.1` as the commit message describes: https://github.com/rails/rails/pull/36029/commits/12a9664ff60f0e2712fd1f79f8dbec06e2f004a2